src/steam_network/user_info_cache.py

```diff
@@ -31,29 +31,31 @@ def is_initialized(self) -> bool:
 
 
     def to_dict(self):
-        creds = {
-            'steam_id': base64.b64encode(str(self._steam_id).encode('utf-8')).decode('utf-8'),
-            'refresh_token': base64.b64encode(str(self._refresh_token).encode('utf-8')).decode('utf-8'),
-            'account_username': base64.b64encode(str(self._account_username).encode('utf-8')).decode('utf-8'),
-            'persona_name': base64.b64encode(str(self._persona_name).encode('utf-8')).decode('utf-8'),
-        }
+        creds = {}
+        if self.is_initialized():
+            creds = {
+                'steam_id': base64.b64encode(str(self._steam_id).encode('utf-8')).decode('utf-8'),
+                'refresh_token': base64.b64encode(str(self._refresh_token).encode('utf-8')).decode('utf-8'),
+                'account_username': base64.b64encode(str(self._account_username).encode('utf-8')).decode('utf-8'),
+                'persona_name': base64.b64encode(str(self._persona_name).encode('utf-8')).decode('utf-8'),
+            }
         return creds
```

Comment:
```suggestion
        return {
                'steam_id': base64.b64encode(str(self._steam_id).encode('utf-8')).decode('utf-8'),
                'refresh_token': base64.b64encode(str(self._refresh_token).encode('utf-8')).decode('utf-8'),
                'account_username': base64.b64encode(str(self._account_username).encode('utf-8')).decode('utf-8'),
                'persona_name': base64.b64encode(str(self._persona_name).encode('utf-8')).decode('utf-8'),
        } if self.is_initialized() else {}
```

just style suggestion feel free to ignore

Reply:
your suggestion might be more pythonic (it's also legal in most languages, in whatever format they prefer) but i personally don't like it because it doesn't resolve until the last line and therefore it's something that could be missed. personal preference i guess, and since i wrote the code i get the final say, XD